### ISSUE TITLE
postscript fails on windows when installing starbird-cli 

### ISSUE THREAD

**Issue by gabriel-myob** [MYOB-Technology/starbird-issues#11]
Hello,

I having issues to install **starbird-cli** on Windows.
Defining **_postinstall_** script as follow solves the issue.

```
...
"postinstall": "node /update/console.js",
...
```
I will raise a pull request with the fix, if that is ok.

**Comment by richard-lopes** [comment]
Not surprised it has some rough edges on Windows at this stage. Thanks @gabriel-myob  and we will make sure it doesn't break anything for MacOSX / Linux users then merge.

**Comment by richard-lopes** [comment]
What kind of issues do you get in more details?

**Comment by gabriel-myob** [comment]
When installing on Windows, the way "postinstall" is defined atm, results in:
```
update/console.js
'update' is not recognized as an internal or external command
```
Prepending "node" ( node update/console.js" ) fix it.

More details on the log file attached.
[postinstall-error.txt](https://github.com/MYOB-Technology/starbird-issues/files/1693417/postinstall-error.txt)

**Comment by richard-lopes** [comment]
I see. Ok. All good 👍

**Comment by richard-lopes** [comment]
Fixed in release v0.2.6
